napari/layers/points/points.py

```diff
@@ -856,6 +868,7 @@ def current_size(self, size: Union[None, float]) -> None:
                 self.size[i, :] = (self.size[i, :] > 0) * size
             self.refresh()
             self.events.size()
+        self._clear_extent_agumented()
```

Comment:
I think this only be in the `if` block above, likely before the refresh because `current_size` only changes `size` or `view_size`.